TextRPG_Team25/Battle.cs

```diff
@@ -40,7 +41,52 @@ private void PlayerAttack(Monster selected)
                 Console.WriteLine($"HP {currentHp}-> HP {selected.HP}");
             }
 
-            
+  
         }
+
+        private void SpawnEnemy() // 몬스터 소환
+        {
+            Random rand = new Random();
+            int spawnNum = rand.Next(1, 4);
+            RanMonsterSpawn(spawnNum);
+        }
+
+        private void RanMonsterSpawn(int spawnNum) // 랜덤 몬스터 소환
+        {
+            Random rand = new Random();
+            for (int i = 0; i < spawnNum; i++)
+            {
+                int num = rand.Next(0, 3);
+                monster.FieldMonster.Add(monster[num]);
+            }
+        }
+
+        public void StartBattle() // 배틀 시작
+        {
+            SpawnEnemy();
+            for(int i = 0; i < monster.FieldMonster.Count; i++)
+            {
+                Console.WriteLine($"Lv.{monster.FieldMonster[i].level}  {monster.FieldMonster[i].name}  HP  {monster.FieldMonster[i].hp}");
+            }
+
+            Console.WriteLine("");
+            Console.WriteLine("[내정보]");
+            Console.WriteLine($"Lv.{player.level}  {player.name} ({player.job})");
+            Console.WriteLine($"HP {player.maxHp}/{player.hp}\n");
+            Console.WriteLine("0. 취소\n");
+            Console.WriteLine("대상을 선택해주세요.\n");
+            Console.WriteLine(">>");
```

Comment:
">>" 옆에 사용자가 입력하도록 하려면 Console.Write()을 사용해야합니다! WriteLine은 줄바꿈을 포함하고 있어여

```suggestion
            Console.Write(">> ");
```